backend/src/test/java/com/javahelp/backend/data/DynamoDBSurveyResponseStoreTest.java

```diff
@@ -0,0 +1,131 @@
+package com.javahelp.backend.data;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+
+import com.amazonaws.regions.Regions;
+import com.javahelp.model.survey.Survey;
+import com.javahelp.model.survey.SurveyQuestion;
+import com.javahelp.model.survey.SurveyQuestionResponse;
+import com.javahelp.model.survey.SurveyResponse;
+import com.javahelp.model.user.ClientUserInfo;
+import com.javahelp.model.user.User;
+
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class DynamoDBSurveyResponseStoreTest {
+    String tableName = "javahelpBackendFilledSurveys";
+
+    Regions regions = Regions.US_EAST_1;
+
+    DynamoDBSurveyResponseStore db = new DynamoDBSurveyResponseStore(tableName, regions);
+
+    @Test(timeout = 5000)
+    public void testCreateRead() {
+        this.db.cleanTable();
+
+        User user = setupUser();
+        SurveyResponse sr = setupSurveyResponse();
+
+        sr = this.db.create(user.getStringID(), sr);
+
+        SurveyResponse read = this.db.read(sr.getID());
+
+        testSurveyEqual(sr.getSurvey(), read.getSurvey());
+        testResponsesEqual(sr.getSurvey(), sr, read);
+
+        this.db.delete(sr.getID());
```

Comment:
Similarly, it might be wise to enclose this in the `finally` block of a `try`-`finally` for guaranteed resource cleanup